### PR TITLE
Handle non-numeric page numbers

### PR DESCRIPTION
**Pull request by timbarkerSE**
This was picked up in fuzzing tests against our site running your plugin. The code errors if a non-numeric page number is supplied. This fix means this (admittedly unlikely) eventuality is handled more elegantly.

### CODE REVIEW

**Review by jrief — COMMENTED** [review]
just for my understanding, how can this bug be reproduced?

File: adminsortable2/admin.py [hunk 1]
```diff
@@ -159,7 +159,10 @@ def get_actions(self, request):
         if len(paginator.page_range) > 1 and 'all' not in request.GET and self.enable_sorting:
             # add actions for moving items to other pages
             move_actions = ['move_to_exact_page']
-            cur_page = int(request.GET.get('p', 0)) + 1
+            try:
+                cur_page = int(request.GET.get('p', 0)) + 1
+            except ValueError:
+                return actions
```
[the remark this answers is not in the document]
Comment (timbarkerSE) [reply adminsortable2/admin.py:165]: Sorry for the delay in replying. To recreate, go to any admin page which uses the SortableAdminMixin and has > 1 page of results. Append to the URL a non-numeric page number e.g. ?p=aaa

I will modify the code to set cur_page = 1 as per your recommendation.

[the remark this answers is not in the document]
Comment (jrief) [reply adminsortable2/admin.py]: Please try version 2.0.x. Lots of things changed there.

**Comment by jrief** [comment]
closed via #321

**Pull request closed without merging**